docs/modules/concepts/pages/user/index.adoc

```diff
@@ -0,0 +1,74 @@
+= James User Model
+:navtitle: User Model
+
+
+For clarity, we have defined various roles for actors who interact 
+with James in one way or another.
```

Comment:
who interact with James **servers**.

We  need to keep some rooms for some other personna like libraryt users (let's keep it for later)

Reply:
Updated. Will push in a few minutes.